ads/inabox/position-observer.js

```diff
@@ -68,9 +72,20 @@ export class PositionObserver {
     }
     // Send the 1st ping immediately
     callback(this.getPositionEntry_(element));
-    this.positionObservable_.add(() => {
+    const positionObservationHandler = (() => {
       callback(this.getPositionEntry_(element));
     });
+    this.positionObservable_.add(positionObservationHandler);
```

Comment:
Add actually returns UnlistenDef so would be cleaner to keep a reference to that than the handler

Reply:
done; that's much nicer